website/templates/project/view_file.mako

```diff
@@ -51,6 +51,7 @@
   <div id="fileViewPanelLeft" class="col-sm-9 panel-expand">
     <div class="row">
       <div id="mfrIframeParent" class="col-sm-9">
+        <p class="text-muted">View this file on <a href=${urls['external']} target="_blank">${provider}</a>.</p>
```

Comment:
`provider` is not human-friendly

![osf___hh_nlp_talk_draft_pdf](https://cloud.githubusercontent.com/assets/2379650/8463559/53ebafe2-2009-11e5-9a48-afe2dfd05efc.png)

We should display the addon's human-friendly representation defined in each addon's config as `FULL_NAME`.